src/cc_catalog_airflow/dags/util/loader/sql.py

```diff
@@ -273,17 +337,15 @@ def _merge_jsonb_arrays(column: str) -> str:
           {_newest_non_null(col.LANDING_URL)},
           {_newest_non_null(col.DIRECT_URL)},
           {_newest_non_null(col.THUMBNAIL)},
-          {_newest_non_null(col.WIDTH)},
-          {_newest_non_null(col.HEIGHT)},
           {_newest_non_null(col.FILESIZE)},
           {_newest_non_null(col.LICENSE)},
           {_newest_non_null(col.LICENSE_VERSION)},
           {_newest_non_null(col.CREATOR)},
           {_newest_non_null(col.CREATOR_URL)},
           {_newest_non_null(col.TITLE)},
-          {_newest_non_null(col.WATERMARKED)},
```

Comment:
We left the `watermarked` field generalized in the [API models](https://github.com/WordPress/openverse-api/blob/c19234fb641a277595f9bcfd04ad4ca4422feadc/openverse-api/catalog/api/models/media.py#L30).

That is how I recently learned that audios can also be watermarked ℹ️

Reply:
I didn't know that. Thank you for this comment! I've made 'watermarked' field a common field for all media types.